media-api/app/lib/elasticsearch/SearchFilters.scala

```diff
@@ -44,7 +44,7 @@ trait SearchFilters extends ImageFields {
   val nonFreeFilter = freeFilter.map(filters.not)
 
   lazy val freeToUseCategories: List[String] =
-    UsageRights.all.filter(ur => ur.defaultCost.contains(Free)).map(ur => ur.category)
+    UsageRights.all.filter(ur => ur.defaultCost.exists(cost => cost == Free | cost == Conditional)).map(ur => ur.category)
```

Comment:
Surprising to use `|` instead of `||` but it turns out that's correct too (though forces evaluation of all operands).